tests/providers/aws/aws_e2e.py

```diff
@@ -15,14 +15,29 @@
 """End to end test for the aws module."""
 import typing
 import unittest
+import warnings
 
 import botocore
 
-from libcloudforensics.providers.aws.internal.common import LOGGER, EC2_SERVICE
+from libcloudforensics.providers.aws.internal.common import EC2_SERVICE
 from libcloudforensics.providers.aws.internal import account
 from libcloudforensics.providers.aws import forensics
+from libcloudforensics import logging_utils
 from tests.scripts import utils
 
+logger = logging_utils.GetLogger(__name__)
+
+
+@typing.no_type_check
+def IgnoreWarnings(test_func):
+  """Disable logging of warning messages."""
+  def DoTest(self, *args, **kwargs):
+    with warnings.catch_warnings():
+      warnings.simplefilter("ignore", ResourceWarning)
+      warnings.simplefilter("ignore", UserWarning)
+      test_func(self, *args, **kwargs)
+  return DoTest
```

Comment:
Why do we do this?

Reply:
This makes logging very ugly because it just floods the console with resource warnings about unclosed ssl sockets etc.
Thus it's really hard to see the logged messages which carry the information that you actually care about.

Reply:
Fair enough. Can you expand the docstring to add a bit more detail about what warnings we're trying to suppress?

Reply:
Done